docs/intro.md

```diff
@@ -0,0 +1,115 @@
+# Introduction
+
+Netlify CMS is a Content Management System for static sites, allowing collaborators to create, edit, review, and publish content without writing code or dealing with version control. It brings the ease of WordPress-style editing to the simplicity and speed of static sites.
+
+At its core, Netlify CMS is an open-source React app that acts as a wrapper for the Git workflow, using the GitHub API. This allows for many advantages, including:
+
+* **Fast, web-based UI** - with rich-text editing, real-time preview, and drag-and-drop media uploads
+* **Platform agnostic** - works with most static site generators
+* **Easy installation** - add two files to your site and hook up the backend by including in your build process or linking to our CDN
+* **Modern authentication** - using GitHub and JSON web tokens
+* **Flexible content types** - specify an unlimited number of content types with custom fields
+* **Fully extensible** - create custom-styled previews, UI widgets, and editor plugins
+
+# Core Concepts
+
+## The Admin Interface
+
+The admin interface is a single-page app with the entry point stored in a static `/admin` folder you add to the root of your site. You can include it with a simple `index.html` file that loads the necessary CSS and JS files from from a CDN:
+
+``` html
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Content Manager</title>
+  
+  <link rel="stylesheet" href="https://unpkg.com/netlify-cms@^0.3/dist/cms.css" />
+  
+</head>
+<body>
+  <script src="https://unpkg.com/netlify-cms@^0.3/dist/cms.js"></script>
+</body>
+</html>
+```
+
+The JS is also available via npm and can be integrated into your regular build process.
+
+### Editorial Workflow
+
+Netlify CMS has an optional [editorial workflow](https://github.com/netlify/netlify-cms/blob/master/docs/editorial_workflow.md) that translates common Git commands into familiar language in a simple UI:
+
+Actions in Netlify UI...	| Perform these Git actions
+--- | ---
+Save draft | Creates a branch and pushes a commit to it
+Edit draft | Pushes another commit to the draft branch
+Mark draft as Waiting for Review | Opens a pull request
+Approve and publish draft | Merges pull request and deletes branch
+
+If you prefer to use Git, you can push additional commits and open and merge PRs directly, and the Netlify CMS interface will update to match.
```

Comment:
Yeah, this only made sense when I thought draft saving didn't create a PR. Rather than get into the details, I'm now thinking I'll just remove the line.